public/index.html

```diff
@@ -11,7 +11,31 @@
 <main>
   <img src='/quote.png' alt='Dev Punks'>
 
+  <section id='mission'>
+  <p>The devPunk subculture, includes a diverse array of ideologies, fashions and forms of expression, including visual art, literature, and film. The subculture is largely characterized by <strong><u>anti-establishment</u> views and the promotion of <u>individual freedom</u></strong>. The devPunk subculture is centered on a loud, aggressive genre of developers. It is usually played by small teams consisting of a junior, one or two seniors, and a hiring manager.
```

Comment:
OHH I see what you did there @mcantor. It's in a way making a noun a verb and vice versa. _(see Google)_ #FunFact: They often call themselves _Googles_ on campus.